experiments/grid_search/loss_map.jl

```diff
@@ -0,0 +1,115 @@
+using Distributed
+@everywhere using Pkg
+@everywhere Pkg.activate(dirname(dirname(@__DIR__)))
+
+@everywhere begin
+    using CalibrateEDMF
+    using CalibrateEDMF.HelperFuncs
+    using CalibrateEDMF.ReferenceStats
+    using CalibrateEDMF.LESUtils
+    using Combinatorics
+    import NCDatasets
+    const NC = NCDatasets
+    config_path = "config.jl"
+    include(config_path)
+
+    function compute_loss_map(
+        # config_path,
+        sim_type = "reference",
+        sims_path = "output/20220328_bNQ/",
+    )
+        config = get_config()
+        n_ens = get_entry(config["grid_search"], "ensemble_size", nothing)
+        case_names = get_entry(config[sim_type], "case_name", nothing)
+
+        # param_paths = filter(isdir, readdir(sims_path, join=true))
+        parameters = get_entry(config["grid_search"], "parameters", nothing)
+        param_names = collect(keys(parameters))
+        NC.Dataset(joinpath(sims_path,"loss_hypercube.nc"),"c") do ds
```

Comment:
```suggestion
        NC.Dataset(joinpath(sims_path,"loss_slices_2D.nc"),"c") do ds
```
? It is not really a hypercube, just 2D slices of it.